lua/telescope/builtin/init.lua

```diff
@@ -382,7 +378,6 @@ builtin.lsp_code_actions = require_on_exported_call("telescope.builtin.lsp").cod
 
 --- Lists any LSP actions for a given range, that can be triggered with `<cr>`
 ---@param opts table: options to pass to the picker
----@field timeout number: timeout for the sync call (default: 10000)
```

Comment:
Good call removing all this invalid docs. Thanks but `timeout` is still valid for `lsp_range_code_actions` because of same underlying function